R/gh_response.R

```diff
@@ -19,7 +23,11 @@ gh_process_response <- function(response) {
   attr(res, "method") <- response$request$method
   attr(res, "response") <- headers(response)
   attr(res, ".send_headers") <- response$request$headers
-  class(res) <- c("gh_response", "list")
+  if(is_raw) {
```

Comment:
Can you please put a space after the `if`?

Reply:
Done